src/components/content/order/OrderSubmit.tsx

```diff
@@ -209,6 +228,10 @@ function OrderSubmit(props: OrderSubmitProps): JSX.Element {
                 <div className={'services-content'}>
                     <div className={'content-title'}>
                         Service: {props.name}@{props.version}
+                        <button className={'content-title-api'} onClick={onclick}>
+                            API DOCUMENT

```

Comment:
Text can be `API Documentation`
And please also try other icons - <ApiOutlined />, <LinkOutlined />,

Reply:
OK，I will modify